doc-content/enterprise-only/fuse/fuse-con.adoc

```diff
@@ -13,10 +13,9 @@ ifdef::DM[]
 Business assets such as rules, decision tables, and DMN models are organized in projects and stored in the {CENTRAL} repository. This ensures consistency, transparency, and the ability to audit across the business. Business users can modify business logic without requiring assistance from IT personnel.
 endif::[]
 
-//You can install {FUSE_LONG} on the Apache Karaf container platform or {EAP_LONG} and then install and configure {PRODUCT} in that container.
-// RESTORE KARAF WHEN SUPPORTED
+You can install {FUSE_LONG} on the Apache Karaf container platform or {EAP_LONG} and then install and configure {PRODUCT} in that container.
```

Comment:
Please remove EAP from this line because Fuse 7.9 and RHDM/RHPAM 7.11 are supported on different EAP versions (7.4 and 7.3)
```suggestion
You can install {FUSE_LONG} on the Apache Karaf container platform and then install and configure {PRODUCT} in that container.
```